libgearman-server/plugins/queue/redis/queue.cc

```diff
@@ -124,6 +126,17 @@ gearmand_error_t Hiredis::initialize()
       "Could not connect to redis server: %s", _redis->errstr);
   }
 
+  if (password.size())
+  {
+	  redisReply *reply = (redisReply*)redisCommand(_redis, "AUTH %s", password.c_str());
+	  if (reply == NULL)
+	  {
+		  return gearmand_gerror("Could not auth with redis server", GEARMAND_QUEUE_ERROR);
+	  }
+	  gearmand_log_debug(GEARMAN_DEFAULT_LOG_PARAM, "hires auth reply: %.*s", (uint32_t)reply->len, reply->str);
```

Comment:
These is more of an 'info' than a debug. As long as we trust Redis not to put sensitive info back into the reply.